<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ShishirDwivedi" name="TestTransaction CreditCard" tests="1" failures="1" timestamp="11 Nov 2014 06:37:42 GMT" time="69.557" errors="0">
  <testcase name="test_CC_AXIS" time="63.696" classname="Test.AdminPanel.Payments.TestTransactionCC">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Test.AdminPanel.Payments.TestTransactionCC.test_CC_AXIS(TestTransactionCC.java:472)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- test_CC_AXIS -->
</testsuite> <!-- TestTransaction CreditCard -->
